app/controllers/users.js

```diff
@@ -0,0 +1,66 @@
+const bcrypt = require('bcryptjs');
+
+const { User } = require('../models');
+const errors = require('../middlewares/errors');
+const logger = require('../logger');
+
+// Verifica la validez de la contraseña (alfanumérica > 7)
+const passwordLength = password => {
+  const characters = /^[a-z0-9]+$/i;
+  const alfanumeric = characters.test(password);
+  const isValid = password.length > 7 && alfanumeric;
+  return isValid;
+};
+
+// Hashea el password mediante bcrypt
+const hashPassword = async password => {
+  const salt = await bcrypt.genSaltSync(10);
+  const hashedPass = await bcrypt.hashSync(password, salt);
+  return hashedPass;
+};
+
+// Verifica la validez y el dominio del email
+const emailValidation = email => {
```

Comment:
What happen if the sended email is...

1. "        @wolox.com.ar"
2. "@wolox.com.ar      "
3. "    @wolox.com.ar    "